test_fms/fms2_io/test_io_simple.F90

```diff
@@ -0,0 +1,90 @@
+!***********************************************************************
+!*                   GNU Lesser General Public License
+!*
+!* This file is part of the GFDL Flexible Modeling System (FMS).
+!*
+!* FMS is free software: you can redistribute it and/or modify it under
+!* the terms of the GNU Lesser General Public License as published by
+!* the Free Software Foundation, either version 3 of the License, or (at
+!* your option) any later version.
+!*
+!* FMS is distributed in the hope that it will be useful, but WITHOUT
+!* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+!* FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+!* for more details.
+!*
+!* You should have received a copy of the GNU Lesser General Public
+!* License along with FMS.  If not, see <http://www.gnu.org/licenses/>.
+!***********************************************************************
+
+! This is a very simple test of FMS IO.
+
+! Ed Hartnett, 6/10/20
+
+program test_io_simple
+  use, intrinsic :: iso_fortran_env, only : real32, real64, int32, int64, error_unit, output_unit
+  use mpi
+  use fms2_io_mod
+  use netcdf_io_mod
+  use mpp_domains_mod
+  use mpp_mod
+  use setup
```

Comment:
Where is this module?  Is it included somewhere?

Reply:
Yes, it is a module and .F90 file in this test directory. I did not add it, I am using it to get going.

Actually I would like to eliminate as much of this code as possible in future iterations of the code. To have a complex test set up, including command line parameters, just makes debugging harder. It would be easier if each test program could stand alone, without shared setup. But I don't have a good grasp yet of all the bits and pieces that FMS requires me to have in order to exercise the I/O layer.

Reply:
OK, thanks for letting me know.  I am not as familiar with the IO tests.